pavelib/assets.py

```diff
@@ -794,6 +796,7 @@ def watch_assets(options):
 
     themes = get_parsed_option(options, 'themes')
     theme_dirs = get_parsed_option(options, 'theme_dirs', [])
+    wait = float(get_parsed_option(options, 'wait', [unicode(DEFAULT_OBSERVER_TIMEOUT)])[0])
```

Comment:
This is a bit terse:
```
[unicode(DEFAULT_OBSERVER_TIMEOUT)])[0]
```

I'm concerned it will be hard to troubleshoot later. Could you make this a multiline with a comment tossed in to describe it?